oh_staff_ui/models.py

```diff
@@ -287,3 +287,61 @@ class ItemCopyrightUsage(models.Model):
     type = models.ForeignKey(
         CopyrightType, on_delete=models.PROTECT, blank=False, null=False
     )
+
+
+class RelationType(models.Model):
+    type = models.CharField(max_length=256, blank=False, null=False)
+
+    def __str__(self):
+        return self.type
+
+
+class Relation(models.Model):
+    value = models.CharField(max_length=1024, blank=False, null=False)
+
+    def __str__(self):
+        return self.value
+
+
+class ItemRelationUsage(models.Model):
+    item = models.ForeignKey(
+        ProjectItem, on_delete=models.PROTECT, blank=False, null=False
+    )
+    relation = models.ForeignKey(
+        Relation, on_delete=models.PROTECT, blank=False, null=False
+    )
+    type = models.ForeignKey(
+        RelationType, on_delete=models.PROTECT, blank=False, null=False
+    )
+
+
+class Format(models.Model):
+    value = models.CharField(max_length=1024, blank=False, null=False)
+
+    def __str__(self):
+        return self.value
+
+
+class ItemFormatUsage(models.Model):
+    item = models.ForeignKey(
+        ProjectItem, on_delete=models.PROTECT, blank=False, null=False
+    )
+    format = models.ForeignKey(
+        Format, on_delete=models.PROTECT, blank=False, null=False
+    )
+
+
+class Coverage(models.Model):
```

Comment:
Discussing in Slack but noting here:

Is there a controlled list of Coverage values that are used?  The legacy app has UI element "Chronological period covered", which seems to be free text and single occurrence, I don't see a way to add more though maybe it repeats after the first is added?

If single-use, maybe this should become an optional attribute of ProjectItem.  If repeatable but uncontrolled, maybe we should just add ProjectItem as a FK in the Coverage table, without an ItemCoverageUsage many-to-many table.

After discussion, since this is repeatable uncontrolled free text, we don't need the Usage table, just add item as FK to `Coverage`.

Reply:
Will add `Coverage` field directly to `ProjectItem`, this is an uncontrolled non-repeatable field